app/views/shared/_personal_key.html.erb

```diff
@@ -26,8 +26,8 @@
   <%= image_tag(asset_url('alert/icon-lock-alert-important.svg'), alt: '', size: '80') %>
 </div>
 <div class="sm-col sm-col-10">
-  <div class="bold margin-bottom-0">
-    <%= t('instructions.personal_key.email_title') %>
+  <div class="margin-bottom-0">
+    <p><strong><%= t('instructions.personal_key.email_title') %></strong></p>
   </div>
```

Comment:
Could we consider using `<p>` in place of the `<div>` if it can serve as a more semantic alternative? Both are block elements, so I expect it could be an easy swap.

```suggestion
  <p class="margin-bottom-0">
    <strong><%= t('instructions.personal_key.email_title') %></strong>
  </p>
```

Reply:
Yes, this is a good idea

Reply:
Might want to check on margins though, since the paragraph tag might add some that didn't exist previously (specifically top? haven't tested).